src/drive/web/modules/selection/duck.js

```diff
@@ -12,21 +9,21 @@ const SHOW_SELECTION_BAR = 'SHOW_SELECTION_BAR'
 const HIDE_SELECTION_BAR = 'HIDE_SELECTION_BAR'
 
 // selectors
-export const isSelected = (state, id) =>
-  state.selection.selected.find(_id => _id === id) !== undefined
-export const getSelectedIds = state => state.selection.selected
+export const isSelected = (state, file) =>
+  state.selection.selected.find(({ id }) => id === file.id) !== undefined
```

Comment:
It seems like the selected is a list. It seems like changing it to be a hash would improve the perf of the existence checks into O(1).